CAIPs/caip-18.md

```diff
@@ -0,0 +1,69 @@
+---
+caip: 18
+title: NFT ID Specification
+author: Joel Thorstensson (@oed)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/18
+status: Draft
+type: Standard
+created: 2020-04-23
+updated: 2020-04-23
+requires: 2
+---
+
+## Simple Summary
+
+CAIP-18 defines a way to identify a specific NFT token on any blockchain specified by CAIP-2 blockchain id.
+
+## Abstract
+
+This proposal aims to facilitate specifying NFT tokens on any blockchain extending CAIP-2 blockchain id specification. This is useful for both decentralized applications and wallets to specify NFT tokens on multiple chains using string identifiers specific to each chain. Currently wallets are usually designed for each chain and multi-chain wallets use proprietray data structures to differentiate NFTs. This proposal aims to standardize these identifiers for NFTs to allow interoperability.
+
+## Motivation
+
+The motivation for proposal stem from designing a chain-agnostic protocol for communication between dapps and wallets that was independent of any blockchain but provide the flexibility to be backwards compatible with existing applications.
+
+## Specification
+
+The NFT id specification will be prefixed with the NFT token ID and delimited with an at sign (`@`) followed by a CAIP-2 blockchain ID.
+
+### Syntax
+
+The `account_id` is a case-sensitive string in the form
+
+```
+account_id:        nft_id + "@" + chain_id
+nft_id:            contract_address + ":" + token_id
+contract_address:  [a-zA-Z0-9]{1,63}
+token_id:          [0-9]{1,78}
+chain_id:          [:-a-zA-Z0-9]{5,64}
+```
```

Comment:
Let's not use it as account_id since it's already used for CAIP-10, I would change this to be called `nft_id` instead. Additionally this is very similar to `account_id` from CAIP-10 so I would change the order of `contract_address` and `token_id` to make use of the existing standard. 

```
nft_id:          token_id + ":" + account_id
account_id:      [@:-a-zA-Z0-9]{6,128}
token_id:        [0-9]{1,78}
```

Reply:
Cool, yeah that makes sense to me 👍